Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -114,246 +114,209 @@ extension PackageShow {
 
 
 extension PackageShow.Model {
-    func authorsClause() -> Node<HTML.BodyContext>? {
```

Comment:
It's great that these are non-optional now! I think we had those because I didn't know about `.empty` early on.